### PR TITLE
Enabled scope services to support Red Hat

### PR DESCRIPTION
**Pull request by tgoettlicher**

### CODE REVIEW

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
( Success: Merged bd5bd571b324b45460fae14209a5c8ca10259f75 into 505dfe4042ed8d49efe39d4deb5426506d6b4a60 )

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
( Success: Merged bd5bd571b324b45460fae14209a5c8ca10259f75 into d5126d6c5c772df1d54b3c9f95f153c998c35299 )

File: plugins/inspect/services_inspector.rb [hunk 1]
```diff
@@ -17,15 +17,15 @@
 
 class ServicesInspector < Inspector
   def inspect(system, description, options = {})
-    if has_systemd(system)
+    if system.has_command?("systemctl", "--version")
```
Comment (thardeck) [inline plugins/inspect/services_inspector.rb]: this is wrong, you only need to provide the command name ```has_command?("systemctl")```

**Comment by thardeck** [comment]
And the branch is not merged against master so has_command is missing.

**Comment by suse-tests-fail** [comment]
Prophet reports failure.
http://ci.opensuse.org/job/machinery-prophet/226464/console
If the given link has expired, you can force a Prophet rerun by just deleting this comment.
( Failure: Merged ee340aef73e7764183fcda7a6c6adabb03245481 into d5126d6c5c772df1d54b3c9f95f153c998c35299 )

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
( Success: Merged 3bdc6ff10441eaa73a81c0f169a58c967c48f1b1 into d5126d6c5c772df1d54b3c9f95f153c998c35299 )

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
( Success: Merged 3bdc6ff10441eaa73a81c0f169a58c967c48f1b1 into d0c0080040fc28087d7e699ff6218a6fddecf9c6 )

**Comment by thardeck** [comment]
I am not so happy about some code parts but since it was reviewed by others and the suggestions were incorporated lgtm

**Pull request merged**